deploy/helm/cloudcost-exporter/README.md

```diff
@@ -0,0 +1,63 @@
+# Cloud Cost Exporter Helm chart
+
+> [!WARNING]
+> This Helm chart is still a WIP.
+> The instructions below are not functional yet.
+>
+> Progress on the Helm chart is tracked here: https://github.com/grafana/cloudcost-exporter/issues/392
```

Comment:
This ^ is the only addition after https://github.com/grafana/cloudcost-exporter/pull/398